packages/storybook-react/src/stories/HeadingGroup.stories.tsx

```diff
@@ -10,37 +11,45 @@ const meta = {
   title: 'React Component/Heading Group',
   id: 'react-heading-group',
   component: HeadingGroup,
-  tags: ['autodocs'],
+  args: {
+    children: (
+      <HeadingGroup style={{ '--utrecht-space-around': 1 } as any}>
+        <Heading1>The Quick Brown Fox Jumps Over The Lazy Dog</Heading1>
+        <Paragraph>The Quick Brown Fox Jumps Over The Lazy Dog</Paragraph>
+      </HeadingGroup>
+    ),
+  },
   parameters: {
     tokensPrefix: 'utrecht-heading-group',
     tokens,
     tokensDefinition,
     docs: {
-      description: {
-        component: readme,
-      },
+      page: () => (
+        <>
+          <Description>{readme}</Description>
+          <Primary />
+          <ArgsTable story={PRIMARY_STORY} />
+          <Stories />
+        </>
+      ),
     },
   },
-} as Meta<typeof HeadingGroup>;
+} satisfies Meta<typeof HeadingGroup>;
 
 export default meta;
 
-const Template: StoryObj<typeof HeadingGroup> = () => (
-  <HeadingGroup style={{ '--utrecht-space-around': 1 } as any}>
-    <Heading1>The Quick Brown Fox Jumps Over The Lazy Dog</Heading1>
-    <Paragraph>The Quick Brown Fox Jumps Over The Lazy Dog</Paragraph>
-  </HeadingGroup>
-);
-
-export const Default = Template.bind({});
+type Story = StoryObj<typeof meta>;
+export const Default: Story = {};
 
-const PreHeadingTemplate: StoryObj<typeof HeadingGroup> = () => (
-  <HeadingGroup style={{ '--utrecht-space-around': 1 } as any}>
-    <Heading1>The Quick Brown Fox Jumps Over The Lazy Dog</Heading1>
-    <PreHeading>The Quick Brown Fox Jumps Over The Lazy Dog</PreHeading>
-  </HeadingGroup>
-);
-
-export const WithPreHeading = PreHeadingTemplate.bind({});
+export const WithPreHeading: Story = {
+  args: {
+    children: (
+      <HeadingGroup style={{ '--utrecht-space-around': 1 } as any}>
+        <Heading1>The Quick Brown Fox Jumps Over The Lazy Dog</Heading1>
+        <PreHeading>The Quick Brown Fox Jumps Over The Lazy Dog</PreHeading>
+      </HeadingGroup>
+    ),
```

Comment:
Using `HeadingGroup` inside `children` leads to a duplicate `HeadingGroup` element:

```jsx
<div className="utrecht-document utrecht-theme">
  <HeadingGroup>
    <HeadingGroup
      style={{
        '--utrecht-space-around': 1
      }}
    >
```

Probably more appropriate to remove the duplicate and use a decorator:

```jsx
{
  decorator: [(Story) => <div style={{ '--utrecht-space-around': 1 } as any}>{Story()}</div>]
}
```